CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+If you got here it means you want to dedicate some of your time and help us with making this library better.  
+So first, thank you ❤️
+
+When contributing to this repository, please first discuss the change you wish to make via [issue](https://github.com/wix/react-native-ui-lib/issues/new/choose),
+[Discord channel](https://discord.gg/2eW4g6Z), or any other method with the owners of this repository before making a change. 
+
+## Pull Request Process
+
+1. First make sure the environment is working and synced with master. For installation details go [here](https://github.com/wix/react-native-ui-lib/blob/master/markdowns/getting-started/setup.md#demo-app)
+2. Before submitting a PR we suggest running `npm run prepush` command that verifies our lint, TS and tests were not broken.
+3. Please don't change our PR template.
+- 3.1 In Description add everything that can help the reviewer and help the reviewing process. 
+- 3.2 In the Changelog it's important a message for the library users - this message will be included in our release notes
+4. We try to make the PR review process as quick as possible, please be patient
+
```

Comment:
```suggestion
5. Please don't submit automatic formatting changes to the files as it results in cumbersome code reviewing.
```

Reply:
But we do want them to keep the code formatted according to our guidelines.. no?

Reply:
Yes. but if they auto format it will take their environment's settings, not ours

Reply:
I'm not sure that works... If the user has some other formatting tool configured in his environment, wouldn't it override the project's format configuration?

Reply:
Maybe, I think we'll learn more after a few PRs and see how it goes. 
And then we can add a more relevant message here about formatting..

Reply:
Sound good